src/components/routes.jsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+
+import DeveloperPreview from './developer-preview';
+import Introduction from './introduction';
+import BasicTutorial from './tutorials/basic-tutorial';
+import AdvancedTutorial from './tutorials/advanced-tutorial';
+import Submission from './api-reference/schemas/submission';
+import MeasurementSets from './api-reference/schemas/measurement-sets';
+import Measurements from './api-reference/schemas/measurements';
+import Benchmarks from './api-reference/schemas/benchmarks';
+import Scoring from './api-reference/scoring';
+import Provider from './api-reference/provider';
+import ExampleDocs from './api-reference/example-docs';
+
+const topicsPaths = [
+  {
+    path: '',
+    exact: true,
+    linkText: 'Introduction',
+    component: () => <Introduction />
```

Comment:
Yes - same as above, wasn't working when I first implemented this but this was not the reason.